s2d-oms-mgmt-solution/azuredeploy.json

```diff
@@ -0,0 +1,1847 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "omsLogAnalyticsWorkspaceName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of existing OMS Log Analytics workspace."
+      }
+    },
+    "omsLogAnalyticsWorkspaceLocation": {
+      "type": "string",
+      "allowedValues": [
+        "East US",
+        "West Europe",
+        "Southeast Asia",
+        "Australia Southeast"
+      ],
+      "metadata": {
+        "description": "The region to of existing OMS Log Analytics workspace."
+      },
+      "defaultValue": "West Europe"
+    }
+  },
+  "variables": {
+    "solutionName": "S2D",
+    "solutionId": "[concat(variables('solutionName'), ' [' , parameters('omsLogAnalyticsWorkspaceName'), ']')]",
+    "_solutionVersion": "1.0",
+    "solutionPublisher": "Cloudadministrator.net",
+    "solutionProduct": "[replace(concat(variables('solutionPublisher'),variables('solutionName')),' ','')]",
+    "solutionPromotionCode": "",
+    "solutionDescription": "OMS Solution for monitoring Storage Spaces Direct (S2D).",
+    "solutionAuthor": "stas@outlook.com",
+    "omsLogAnalyticsWorkspaceApiVersion": "2015-11-01-preview"
```

Comment:
No, this is the correct API for this - preview for both OperatinalInsight and OperationsManagement RP.